ang/civicase/case/search/directives/search.directive.js

```diff
@@ -214,6 +279,19 @@
       }
     }
 
+    /**
+     * Returns Select2 values as arrays. Select2 returns a single selected value
+     * as an array, but multiple values as a string separated by comas.
+     *
+     * @param {Array|String} value the value as provided by Select2.
+     * @return {Array}
+     */
+    function getSelect2ValueAsArray (value) {
```

Comment:
I can see that we are already using this kind of logic in other places. so please create a factory/service for this. No need to replace the logic in all places, but at least lets create it so that we can use the same in future.

Reply:
Will do. Can you let me know what those other places are so I can look and see if they do anything differently?

Reply:
There are many actually, if you search with `split(',')` in the whole project, you can find it.